runtime/pom.xml

```diff
@@ -250,4 +237,29 @@
       <version>1.7.10</version>
     </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>fabric8</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <version>${fabric8.maven.plugin.version}</version>
+            <executions>
+              <execution>
```

Comment:
I would leave the configuration outside the profile and only remove the executions or move the executions into a profile. That way people can use goals like 'fabric8:resource' without specifying this profile.